tools/sql_extraction/src/main/kotlin/Cli.kt

```diff
@@ -74,13 +72,28 @@ private class Cli(
         "--pretty", help = "Pretty-print output JSON"
     ).flag()
 
+    private val showProgress: Boolean by option(
+        "--progress", help = "Print progress to STDERR"
+    ).flag()
+
+    private val debug: Boolean by option(
+        "--debug", "--verbose", help = "Print debug logs to STDERR"
+    ).flag()
+
     override fun run() {
-        LOGGER.debug("Starting SQL Extraction from command line")
+        if (debug) {
+            System.setProperty(org.slf4j.impl.SimpleLogger.DEFAULT_LOG_LEVEL_KEY, "Debug")
```

Comment:
This setProperty looks a bit hard coded, do we have to update this if our dependency/libs change in future?

Reply:
Yes, this would have to change if we were to change the logging dependencies from slf4j to something else.

Reply:
I realized I can import SimpleLogger for brevity.